presubmit.py

```diff
@@ -13,10 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Presubmit script for fuzzbench."""
+# pylint: disable=wrong-import-position
+import os
+
+# Many users need this if they are using a Google Cloud instance for development
+# or if their system has a weird setup that makes FuzzBench think it is running
+# on Google Cloud. It's unlikely that setting this will mess anything up so set
+# it.
+# TODO(metzman): Make local the default setting and propagate 'NOT_LOCAL' to all
+# production environments so we don't need to worry about this any more.
+os.environ['FORCE_LOCAL'] = '1'
```

Comment:
what if the user passed FORCE_NOT_LOCAL? Shall we detect that we're using both FORCE_NOT_LOCAL and FORCE_LOCAL and throw an error (since they are mutually exclusive)? Maybe this case never happens in practice?

Reply:
I deleted this, good point.

Reply:
Sorry. I assumed I was replying to your previous comment.

Reply:
I added an assertion to ensure this isn't happening.